src/locale/pages/organize/sv.yml

```diff
@@ -0,0 +1,13 @@
+linkGroup:
+  public: Offentlig sida
+  manage: Hantera organisation
+  settings: Ändra inställningar
+currentProjects:
+  title: Nuvarande projekter och kampanjer
+  all: Alla projekter och kampanjer
```

Comment:
These should be "Nuvarande projekt och kampanjer" and "Alla projekt och kampanjer" respectively.

Adding Swedish translations is not necessary, but do keep them coming if you enjoy it! 😊